src/main/java/com/noroff/mefit/service/SetService.java

```diff
@@ -0,0 +1,52 @@
+package com.noroff.mefit.service;
+
+import com.noroff.mefit.model.Exercise;
+import com.noroff.mefit.model.Set;
+import com.noroff.mefit.repository.SetRepository;
+import org.springframework.stereotype.Service;
+
+import java.util.List;
+
+@Service
+public record SetService(SetRepository setRepository) {
+    public List<Set> getAll() {
+        return setRepository.findAll();
+    }
+
+    public Set create(Set set) {
+        return setRepository.save(set);
+    }
+
+    public Set getById(Long setId) {
+        if (!setRepository.existsById((setId))) {
```

Comment:
Double parentheses around `setId`
```suggestion
        if (!setRepository.existsById(setId)) {
```